ownCloud/Resources/Info.plist

```diff
@@ -29,8 +33,14 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
+	<key>NSAppleMusicUsageDescription</key>
+	<string>This permission is needed by the ownCloud app to upload media files to your server.</string>
 	<key>NSFaceIDUsageDescription</key>
 	<string>Unlock the app without entering the passcode.</string>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+	<string>This permission is needed by the ownCloud app to upload photos and videos from your photo library.</string>
```

Comment:
Same here, sorry.

"This permission is needed to upload photos and videos from your photo library."?